x-pack/plugins/ml/public/components/entity_cell/entity_cell.scss

```diff
@@ -0,0 +1,26 @@
+.field-value-short {
+  @include euiTextTruncate;
+  display: inline-block;
+  vertical-align: bottom;
+}
+
+.field-value-long {
+  overflow-wrap: break-word;
+  // Known issue: IE11 text overflow within a flex container
```

Comment:
Is this comment still needed? The layout looked ok on IE11.

Reply:
It is not. Same as above. Thanks! 😄